static/app/utils/profiling/renderers/cursorRenderer.tsx

```diff
@@ -13,8 +13,6 @@ class CursorRenderer {
     this.canvas = canvas;
     this.theme = theme;
     this.context = getContext(canvas, '2d');
-
-    resizeCanvasToDisplaySize(canvas);
```

Comment:
This was unnecessary (found when profiling) as the canvas was already resized by other renderers